common/locales/load.ts

```
@@ -2,17 +2,10 @@
 import de from './de.json' with { type: 'json' }
 import en from './en.json' with { type: 'json' }
 
-export function loadLocales(overwriteJSONstring?: string) {
+export function loadLocales(overwrite: { [key in Locale]?: { [key: string]: string } }) {
+  delete (overwrite as any)._id
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)